core/utils/poly-look/src/react/index.js

```diff
@@ -0,0 +1,2 @@
+import Chips from "./chips/chips.jsx";
+export { Chips };
```

Comment:
Nit: I thought you can just do something like: `export {Chips} from "./chips/chips.jsx";`